ssl/ssl_lib.cc

```diff
@@ -1455,6 +1438,19 @@ int SSL_get_error(const SSL *ssl, int ret_code) {
     }
   }
 
+  // An EOF was observed which violates the protocol, and the underlying
+  // transport does not participate in the error queue.
+  //
+  // This was moved earlier in the function by BoringSSL and used to be
+  // wrapped with a check for |ret_code| == 0 in OpenSSL 1.0.2. Since
+  // OpenSSL 1.1.1, the check for |ret_code| has been removed and we've
+  // moved this back before the final return to gain better parity with
+  // OpenSSL.
+  // See openssl/openssl@8051ab2 for more details.
+  if (ssl->s3->rwstate == SSL_ERROR_ZERO_RETURN) {
+    return SSL_ERROR_ZERO_RETURN;
+  }
```

Comment:
if we move this back up to where it was, do your bind9 tests still pass?

Reply:
gotcha. we can just bring `SSL_ERROR_ZERO_RETURN` into a case clause on L1397. I don't think we need the comments anymore.

Reply:
Great catch, just moved it to the switch statement